src/NLog/Internal/NetworkSenders/QueuedNetworkSender.cs

```diff
@@ -78,6 +78,7 @@ protected QueuedNetworkSender(string url)
         public int MaxQueueSize { get; set; }
 
         public NetworkTargetQueueOverflowAction OnQueueOverflow { get; set; }
+        public event EventHandler<LogEventDroppedEventArgs> LogEventDropped;
```

Comment:
Using `EventHandler` is dangerous, as you have to reset/unhook them (assign to `null`) or else you will experience "memory leak".

Reply:
From my analysis there is no way for `QueuedNetworkSender`(publisher) to outlive `NetworkTarget`(subscriber), so I can rely on garbage collector to clean up memory.

Reply:
Seems the garbage collector is happily clearning up.